subsys/bluetooth/services/fast_pair/use_case/Kconfig

```diff
@@ -42,6 +42,9 @@ config BT_FAST_PAIR_USE_CASE_LOCATOR_TAG
 	bool "Locator tag use case"
 	select BT_FAST_PAIR_FMDN
 	select BT_FAST_PAIR_FMDN_DULT
+	select BT_FAST_PAIR_USE_CASE_UNSUPPORTED_GATT_SERVICE_MODEL_ID
+	select BT_FAST_PAIR_USE_CASE_UNSUPPORTED_SUBSEQUENT_PAIRING
+	select BT_FAST_PAIR_USE_CASE_UNSUPPORTED_PN
```

Comment:
Looking at https://developers.google.com/nearby/fast-pair/specifications/devicefeaturerequirement/devicefeaturerequirement_locatortags, the `Firmware revision` is marked as `Not Supported`. Should we add `CONFIG_BT_DIS_FW_REV` as an unsupported dependency here too?